styles/components/cards.css

```diff
@@ -0,0 +1,55 @@
+.card{
+    max-width: 200px; 
+    margin: 0.5rem;
+    background-color: var(--white);        
+}
+.card:hover{
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+}
+
+.card-image{
+    width: 100%;    
+}
+.card-content{
+    padding: 0.5rem;
+
+}
+.card-header{
+    font-weight: bold;
+    font-size: 1rem;
+    margin : 0.5rem 0 0 0;
+}
+.card-text{
+    font-size: 0.8rem;
+    margin : 0.5rem 0 0.5rem 0;  
+
+}
+.card-badge{
+    position: relative;
+}
+.card-badge-item{
+    position : absolute;
+    right:-10px;     
+}
+.cancel-svg{
```

Comment:
U can rename like "card-cancel-tag" for better understanding !! If you wish you can change it